pierce-portfolio/portfolio/src/main/webapp/index.html

```diff
@@ -1,14 +1,48 @@
 <!DOCTYPE html>
 <html>
-  <head>
+   <head>
```

Comment:
Can you fix the spacing for this file according to this style guide? https://google.github.io/styleguide/htmlcssguide.html

Specifically, two space indenting.